Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [TerriaJS/TerriaJSNext](https://hosted.weblate.org/projects/terriajs/terriajsnext/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/terriajs/-/terriajsnext/horizontal-auto.svg)

staffordsmith83 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 17.4% (213 of 1218 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/TerriaJS/terriajs?pullRequest=6575) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/TerriaJS/terriajs?pullRequest=6575) before we can accept your contribution.<br/>**1** out of **6** committers have signed the CLA.<br/><br/>:white_check_mark: staffordsmith83<br/>:x: imakihi<br/>:x: feralex12131415<br/>:x: trendspotter<br/>:x: SSiline<br/>:x: MarongHappy<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/TerriaJS/terriajs?pullRequest=6575) it.</sub>

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 18.7% (228 of 1219 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

Commit:
Translated using Weblate (Catalan) for TerriaJSNext

Currently translated at 100.0% (1219 of 1219 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/ca/

imakihi committed:
Translated using Weblate (Japanese) for TerriaJSNext

Currently translated at 100.0% (1219 of 1219 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/ja/

trendspotter committed:
Translated using Weblate (Czech) for TerriaJSNext

Currently translated at 100.0% (1219 of 1219 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/cs/

imakihi committed:
Translated using Weblate (Japanese) for TerriaJSNext

Currently translated at 100.0% (1219 of 1219 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/ja/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 19.0% (232 of 1221 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

Commit:
Translated using Weblate (Catalan) for TerriaJSNext

Currently translated at 100.0% (1221 of 1221 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/ca/

imakihi committed:
Translated using Weblate (Japanese) for TerriaJSNext

Currently translated at 100.0% (1221 of 1221 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/ja/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 21.4% (262 of 1221 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 22.4% (274 of 1221 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 25.6% (311 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

staffordsmith83 reviewed (approved):
Looks good to my limited knowledge

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 25.9% (315 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 28.2% (343 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

imakihi committed:
Translated using Weblate (Japanese) for TerriaJSNext

Currently translated at 100.0% (1213 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/ja/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 29.2% (355 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 39.4% (478 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

MarongHappy committed:
Translated using Weblate (Korean) for TerriaJSNext

Currently translated at 6.4% (78 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/ko/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 41.3% (502 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 41.4% (503 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 42.5% (516 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 43.6% (530 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 43.9% (533 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 46.0% (558 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 46.9% (569 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 47.1% (572 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

trendspotter committed:
Translated using Weblate (Czech) for TerriaJSNext

Currently translated at 100.0% (1213 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/cs/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 48.0% (583 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 48.2% (585 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

SSiline committed:
Translated using Weblate (Khmer (Central)) for TerriaJSNext

Currently translated at 60.5% (734 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/km/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 48.4% (588 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 49.2% (597 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 49.3% (599 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 49.5% (601 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 49.6% (602 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 49.7% (603 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/

feralex12131415 committed:
Translated using Weblate (Spanish) for TerriaJSNext

Currently translated at 50.6% (614 of 1213 strings)

Translation: TerriaJS/TerriaJSNext
Translate-URL: https://hosted.weblate.org/projects/terriajs/terriajsnext/es/
